<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | BioBits by Steve </title> <meta name="author" content="Steve Baeyen"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://stevebaeyen.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> BioBits by Steve </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ade29cbf1df348388b5f351822a87b9d" class="col-sm-8"> <div class="title">Year-long, multiple-timepoint field studies show the importance of spatiotemporal dynamics and microbial functions in agricultural soil microbiomes</div> <div class="author"> Lisa Joos, Sarah Ommeslag, <em>Steve Baeyen</em>, Wouter Asselberg, {Koen Van} Loo, Lieven Clement, Jane Debode, Bart Vandecasteele, and {Caroline De} Tender </div> <div class="periodical"> <em>mSystems</em>, Jul 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1128/msystems.00112-25" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Despite the recognition of the complexity of soil ecosystem dynamics, most soil microbiome studies sample one field, take one sample per field, or use limited samples throughout the year. This limits our understanding of the spatiotemporal role of the soil microbiome in relation to management practices. To address these limitations, we conducted a year-long investigation of the soil microbiome in two agricultural fields, sampling multiple plots at different soil depths every 5 weeks. We examined spatial and temporal variabilities in response to the application of organic amendments (one-time biochar and annual compost application) on bacterial and fungal communities, studying both the microbial composition (metabarcoding, phospholipid fatty acids [PLFA], hot-water extractable-carbon) and activity (metatranscriptomics). Indicated by metabarcoding and PLFA, fungal communities were less affected over time per field, whereas bacteria exhibited more pronounced temporal trends. In contrast, fungi displayed clear spatial effects, while bacterial spatial differences within the field were predominantly observed in the deeper soil layer. Effects on functional roles and metabolic processes of the active microbial community were mainly related to temporal trends, especially in the topsoil. Organic amendments did not affect the microbial activity and affected fewer than 2 emphasizing the importance of field-specific factors, sampling depth, and community type. This ushers in the need for a well-considered experimental design and sampling strategy that accounts for spatiotemporal trends.IMPORTANCE This study addresses a critical gap in soil microbiome research by investigating spatiotemporal effects on soil bacterial and fungal composition and activity in relation to field management practices. Moving beyond single-field and limited sampling approaches, this research conducted monthly sampling events on two fields at various depths. By combining metabarcoding, phospholipid fatty acid analysis, and metatranscriptomics, the study examined bacterial and fungal community composition, biomass, and functionality. Key findings reveal distinct responses of bacterial and fungal communities to spatiotemporal variability and management practices. Functional categories were predominantly driven by temporal trends rather than compost amendments. Temporal changes were more pronounced in the topsoil. These insights into the complex interactions between soil microbial communities, management practices, and spatiotemporal dynamics contribute significantly to soil microbiome research and sampling strategies.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Veltjen_2024" class="col-sm-8"> <div class="title">The “Key” to Bringing DNA Collections to the Next Level: A DiSSCo Flanders Working Group Product</div> <div class="author"> Emily Veltjen, Pieter Asselman, Wim Baert, <em>Steve Baeyen</em>, Lise Beirinckx, Liselot Breyne, Dimitri Brosens, Tim Claerhout, Sari Cogneau, Karen Cox, Laura Cuypers, Lynn Delgat, Philippe Desmeth, Jordi Raad, Lore Esselens, Maria-Rose Eves Down, Frederik Leliaert, Kenny Meganck, Zjef Pereboom, Nathalie Smitz, Gontran Sonet, Maarten Trekels, An Vanden Broeck, Charlotte Van Driessche, and Aaike De Wever </div> <div class="periodical"> <em>Biodiversity Information Science and Standards</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.3897/biss.8.141720" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Veltjen_2025" class="col-sm-8"> <div class="title">The key to bringing DNA collections to the next level</div> <div class="author"> Emily Veltjen, Pieter Asselman, Wim Baert, <em>Steve Baeyen</em>, Lise Beirinckx, Liselot Breyne, Dimitri Brosens, Tim Claerhout, Sari Cogneau, Karen Cox, Laura Cuypers, Lynn Delgat, Philippe Desmeth, Jordi Raad, Lore Esselens, Maria-Rose Eves Down, Philippe Helsen, Frederik Leliaert, Kenny Meganck, Zjef Pereboom, Nathalie Smitz, Gontran Sonet, Maarten Trekels, An Vanden Broeck, Charlotte Van Driessche, and Aaike De Wever </div> <div class="periodical"> <em>Research Ideas and Outcomes</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.3897/rio.10.e135978" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="https://doi.org/10.5281/zenodo.13598099" class="col-sm-8"> <div class="title">Snakemake workflow for Illumina bacterial assembly and QC</div> <div class="author"> <em>Steve Baeyen</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.5281/ZENODO.13598099" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="https://doi.org/10.5281/zenodo.13597370" class="col-sm-8"> <div class="title">Snakemake workflow for Nanopore-only bacterial assembly and QC</div> <div class="author"> <em>Steve Baeyen</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.5281/ZENODO.13597370" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="https://doi.org/10.5281/zenodo.13596661" class="col-sm-8"> <div class="title">Snakemake workflow for bacterial assembly QC</div> <div class="author"> <em>Steve Baeyen</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.5281/ZENODO.13596661" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="https://doi.org/10.5281/zenodo.13596660" class="col-sm-8"> <div class="title">Snakemake workflow for bacterial assembly QC</div> <div class="author"> <em>Steve Baeyen</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.5281/ZENODO.13596660" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2024" class="col-sm-8"> <div class="title">Multiphasic investigations imply transfer of orange-/red-pigmented strains of the bean pathogen Curtobacterium flaccumfaciens pv. flaccumfaciens to a new species as C. aurantiacum sp. nov., elevation of the poinsettia pathogen C. flaccumfaciens pv. poinsettiae to the species level as C. Poinsettiae sp. nov., and synonymy of C. albidum with C. citreum</div> <div class="author"> E. Osdaghi, S.M. Taghavi, M. Hamidizade, M. Kariminejhad, A. Fazliarab, H. Hajian Maleki, <em>S. Baeyen</em>, G. Taghouti, M.-A. Jacques, J. Van Vaerenbergh, and P. Portier </div> <div class="periodical"> <em>Systematic and Applied Microbiology</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="5ef69c3bb7cf452f8f5f980552e85b33" class="col-sm-8"> <div class="title">Presence of Curtobacterium flaccumfaciens in Belgian and Dutch greenhouse poinsettia production – Ruining the Christmas spirit?</div> <div class="author"> Jolien Venneman, M.J.C. Pel, J.L.J. Bilt}, Cinzia {Van Malderghem}, Margo {De Smet}, <em>Steve Baeyen</em>, and Johan {Van Vaerenbergh} </div> <div class="periodical"> Aug 2023 </div> <div class="periodical"> 12th International Congress of Plant Pathology (ICPP) ; Conference date: 20-08-2023 Through 25-08-2023 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="8239e3c167344d73bd9f2c204645dd51" class="col-sm-8"> <div class="title">The fate of Xylella fastidiosa in common woody plant species in Belgium.</div> <div class="author"> Jolien Venneman, Cinzia {Van Malderghem}, Paul Quataert, Brigitte {De Paepe}, Yannick {Van Moerkercke}, <em>Steve Baeyen</em>, and Johan {Van Vaerenbergh} </div> <div class="periodical"> Aug 2023 </div> <div class="periodical"> 4th European Conference on Xylella fastidiosa ; Conference date: 20-08-2023 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2023" class="col-sm-8"> <div class="title">Back to the Roots: Agrobacterium-Specific Phages Show Potential to Disinfect Nutrient Solution from Hydroponic Greenhouses</div> <div class="author"> K.J. Fortuna, D. Holtappels, J. Venneman, <em>S. Baeyen</em>, M. Vallino, P. Verwilt, H. Rediers, B. De Coninck, M. Maes, J. Van Vaerenbergh, R. Lavigne, and J. Wagemans </div> <div class="periodical"> <em>Applied and Environmental Microbiology</em>, Aug 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2025" class="col-sm-8"> <div class="title">First Report of Black Rot Disease in Eruca vesicaria subsp. sativa Caused by Xanthomonas campestris pv. campestris in Belgium</div> <div class="author"> D. Holtappels, B. De Paepe, C. Malderghem, <em>S. Baeyen</em>, J. Venneman, J. Wagemans, and J. Van Vaerenbergh </div> <div class="periodical"> <em>Plant Disease</em>, Aug 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="198a8548d4a8464882e0727afa27e7e9" class="col-sm-8"> <div class="title">Exploring Acidovorax strains causing leaf spot on Calathea.</div> <div class="author"> Johan {Van Vaerenbergh}, Jolien Venneman, Brigitte {De Paepe}, Cinzia {Van Malderghem}, and <em>Steve Baeyen</em> </div> <div class="periodical"> Jul 2022 </div> <div class="periodical"> 14th International Conference on Plant Pathogenic Bacteria (ICPPB) ; Conference date: 03-07-2022 Through 08-07-2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ebd1c6de3750437e9558f9bf765ea874" class="col-sm-8"> <div class="title">Uncommon Curtobacterium flaccumfaciens pathovars revive in other plant hosts: pv. betae in sugar beet and pv. oortii in garden lily.</div> <div class="author"> Johan {Van Vaerenbergh}, Jolien Venneman, Brigitte {De Paepe}, Cinzia {Van Malderghem}, <em>Steve Baeyen</em>, and A. Wauters </div> <div class="periodical"> Jul 2022 </div> <div class="periodical"> 14th International Conference on Plant Pathogenic Bacteria (ICPPB) ; Conference date: 03-07-2022 Through 08-07-2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Holtappels_2022" class="col-sm-8"> <div class="title">The potential of bacteriophages to control Xanthomonas campestris pv. campestris at different stages of disease development</div> <div class="author"> Dominique Holtappels, Kiandro J. Fortuna, Lauren Moons, Nand Broeckaert, Léon E. Bäcker, Sofie Venneman, Sofie Rombouts, Louis Lippens, <em>Steve Baeyen</em>, Sabien Pollet, Jean-Paul Noben, Frank Oechslin, Marta Vallino, Abram Aertsen, Martine Maes, Johan Van Vaerenbergh, Rob Lavigne, and Jeroen Wagemans </div> <div class="periodical"> <em>Microbial Biotechnology</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/1751-7915.14004" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2022" class="col-sm-8"> <div class="title">Facilitating the adoption of high-throughput sequencing technologies as a plant pest diagnostic test in laboratories: A step-by-step description</div> <div class="author"> B. Lebas, I. Adams, M. Al Rwahnih, <em>S. Baeyen</em>, G. Bilodeau, A.G. Blouin, N. Boonham, T. Candresse, A. Chandelier, K. De Jonghe, A. Fox, Y.Z.A. Gaafar, P. Gentit, A. Haegeman, W. Ho, O. Hurtado-Gonzales, W. Jonkers, J. Kreuze, D. Kutjnak, B. Landa, M. Liu, F. Maclot, M. Malapi-Wight, H.J. Maree, F. Martoni, N. Mehle, A. Minafra, D. Mollov, A. Moreira, M. Nakhla, F. Petter, A.M. Piper, J. Ponchart, R. Rae, B. Remenant, Y. Rivera, B. Rodoni, J. Roenhorst, J. Rollin, P. Saldarelli, J. Santala, R. Souza-Richards, D. Spadaro, D.J. Studholme, S. Sultmanis, R. Vlugt, L. Tamisier, C. Trontin, I. Vazquez-Iglesias, C.S.L. Vicente, B.T.L.H. Vossenberg, T. Wetzel, H. Ziebell, and S. Massart </div> <div class="periodical"> <em>EPPO Bulletin</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2026" class="col-sm-8"> <div class="title">Guidelines for the reliable use of high throughput sequencing technologies to detect plant pathogens and pests</div> <div class="author"> S. Massart, I. Adams, M. Al Rwahnih, <em>S. Baeyen</em>, G.J. Bilodeau, A.G. Blouin, N. Boonham, T. Candresse, A. Chandellier, K. De Jonghe, A. Fox, Y.Z.A. Gaafar, P. Gentit, A. Haegeman, W. Ho, O. Hurtado-Gonzales, W. Jonkers, J. Kreuze, D. Kutjnak, B.B. Landa, M. Liu, F. Maclot, M. Malapi-Wight, H.J. Maree, F. Martoni, N. Mehle, A. Minafra, D. Mollov, A.G. Moreira, M. Nakhla, F. Petter, A.M. Piper, J.P. Ponchart, R. Rae, B. Remenant, Y. Rivera, B. Rodoni, M. Botermans, J.W. Roenhorst, J. Rollin, P. Saldarelli, J. Santala, R. Souza-Richards, D. Spadaro, D.J. Studholme, S. Sultmanis, R. Vlugt, L. Tamisier, C. Trontin, I. Vazquez-Iglesias, C.S.L. Vicente, B.T.L.H. Vossenberg, M. Westenberg, T. Wetzel, H. Ziebell, and B.S.M. Lebas </div> <div class="periodical"> <em>Peer Community Journal</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2027" class="col-sm-8"> <div class="title">Promising epiphytic antagonistic Pseudomonas strains from the Citrus phyllosphere in the biocontrol of Pseudomonas syringae pv. syringae, causing bacterial citrus blast and black pit</div> <div class="author"> E. Abdellatif, A. Mikici?ski, M. Ka?u?na, B.A. Bahri, A. Mougou-Hamdane, J. Vaerenberg, <em>S. Baeyen</em>, A. Rhouma, and J.D. Janse </div> <div class="periodical"> <em>Journal of Plant Pathology</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2021" class="col-sm-8"> <div class="title">Effects of bioavailable phosphorus and soil biota on typical Nardus grassland species in competition with fast-growing plant species</div> <div class="author"> S. Schelfhout, S. Wasof, J. Mertens, M. Vanhellemont, A. Demey, A. Haegeman, E. DeCock, I. Moeneclaey, P. Vangansbeke, N. Viaene, <em>S. Baeyen</em>, N. De Sutter, M. Maes, W.H. Putten, K. Verheyen, and A. De Schrijver </div> <div class="periodical"> <em>Ecological Indicators</em>, Jan 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Joos_2020" class="col-sm-8"> <div class="title">Daring to be differential: metabarcoding analysis of soil and plant-related microbial communities using amplicon sequence variants and operational taxonomical units</div> <div class="author"> Lisa Joos, Stien Beirinckx, Annelies Haegeman, Jane Debode, Bart Vandecasteele, <em>Steve Baeyen</em>, Sofie Goormachtig, Lieven Clement, and Caroline De Tender </div> <div class="periodical"> <em>BMC Genomics</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/s12864-020-07126-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Abdellatif_2020" class="col-sm-8"> <div class="title">Phylogenetic, genetic, and phenotypic diversity of Pseudomonas syringae pv. syringae strains isolated from citrus blast and black pit in Tunisia</div> <div class="author"> Emna Abdellatif, Monika Kałużna, Patrizia Ferrante, Marco Scortichini, Bochra Bahri, Jaap D. Janse, Johan Vaerenberg, <em>Steve Baeyen</em>, Piotr Sobiczewski, and Ali Rhouma </div> <div class="periodical"> <em>Plant Pathology</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/ppa.13244" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Onrust_2020" class="col-sm-8"> <div class="title">Effect of in feed administration of different butyrate formulations on Salmonella Enteritidis colonization and cecal microbiota in broilers</div> <div class="author"> Lonneke Onrust, <em>Steve Baeyen</em>, Freddy Haesebrouck, Richard Ducatelle, and Filip Van Immerseel </div> <div class="periodical"> <em>Veterinary Research</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/s13567-020-00780-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Desmet_2019" class="col-sm-8"> <div class="title">Differential efficiency of wild type rhizogenic strains for rol gene transformation of plants</div> <div class="author"> Siel Desmet, Ellen De Keyser, Johan Van Vaerenbergh, <em>Steve Baeyen</em>, Johan Van Huylenbroeck, Danny Geelen, and Emmy Dhooghe </div> <div class="periodical"> <em>Applied Microbiology and Biotechnology</em>, Jul 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1007/s00253-019-10003-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vermeulen_2018" class="col-sm-8"> <div class="title">Reduced-Particle-Size Wheat Bran Is Efficiently Colonized by a Lactic Acid-Producing Community and Reduces Levels of Enterobacteriaceae in the Cecal Microbiota of Broilers</div> <div class="author"> Karen Vermeulen, Joran Verspreet, Christophe M. Courtin, Freddy Haesebrouck, <em>Steve Baeyen</em>, Annelies Haegeman, Richard Ducatelle, and Filip Van Immerseel </div> <div class="periodical"> <em>Applied and Environmental Microbiology</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1128/aem.01343-18" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="G_taz_2018" class="col-sm-8"> <div class="title"> High-Quality Draft Genome Sequences of Five Xanthomonas arboricola pv. fragariae Isolates </div> <div class="author"> Michael Gétaz, <em>Steve Baeyen</em>, Jochen Blom, Martine Maes, Bart Cottyn, and JoÃ«l F. Pothier </div> <div class="periodical"> <em>Genome Announcements</em>, Feb 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1128/genomea.01585-17" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Constantin_2017" class="col-sm-8"> <div class="title">Pathogenicity and virulence gene content of Xanthomonas strains infecting Araceae, formerly known as Xanthomonas axonopodis pv. dieffenbachiae </div> <div class="author"> E. C. Constantin, A. Haegeman, J. Van Vaerenbergh, <em>S. Baeyen</em>, C. Van Malderghem, M. Maes, and B. Cottyn </div> <div class="periodical"> <em>Plant Pathology</em>, Mar 2017 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/ppa.12694" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="46f9ad22db7f4c7e820d8e71427e56b2" class="col-sm-8"> <div class="title">EFFECT OF FOOD-BORNE FUSARIUM MYCOTOXINS ON THE GUT MICROBIOME COMPOSITION IN BROILER CHICKENS AND IN PIGS</div> <div class="author"> Gunther Antonissen, Filip {Van Immerseel}, A. Michiels, Venessa Eeckhaut, K. Vermeulen, N. Reisinger, <em>Steve Baeyen</em>, Annelies Haegeman, K Audenaert, Sarah {De Saeger}, Dominiek Maes, Freddy Haesebrouck, Richard Ducatelle, and Siska Croubels </div> <div class="periodical"> Sep 2017 </div> <div class="periodical"> abstract for the first MycoKey international Conferenceheld in Ghent, Belgium from 11 to 14 September 2017; 1st MycoKey International Conference ; Conference date: 11-09-2017 Through 14-09-2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The mycotoxins deoxynivalenol (DON) and fumonisins (FBs) exert toxicity toward the intestinal epithelial barrier and immune system. Besides, following ingestion of contaminated feed, the intestinal microbiota will be exposed to these toxins. The gut harbours a complex community of over trillions microbial cells which influence animal physiology, metabolism, nutrition and immune function, while disruption of the gut microbiota has been linked with poor intestinal health. The aim of this presentation is to discuss comparatively the impact of mycotoxins on intestinal microbiota composition in broiler chickens and pigs. In a first study, the impact of DON and FBs on the composition of the intestinal microbiota in broiler chickens of different ages was assessed. Chickens of 1 day old were fed either a control diet, a DON contaminated diet (3.3-3.7 mg DON/kg feed) or a FBs contaminated diet (15.5-18.3 mg FB1+FB2/kg feed). Subsequently, after a feeding period of one, three and five weeks chickens were euthanized and ileal and caecal content samples were collected. In a second study, the impact of DON and its acetylated forms on the gut microbiome in piglets of 3 weeks old was investigated. Piglets were fed either a control diet or a DON contaminated diet (0.4 mg DON, 0.3 mg 3-acetylDON, and 0.8 mg 15-acetylDON/kg feed) for 5 weeks. Following euthanasia, colon content samples were collected. Intestinal microbiome profiling was performed by next-generation sequencing of 16S ribosomal DNA. The hyper-variable V4 region was amplified by using the 515F-806R primer set, and the amplicon library was prepared with Nextera adaptors. Subsequently, PCR fragments were sequenced by MiSeq Illumina sequencing (96 samples per lane, 2x300bp 5Gb raw data per lane). The results showed that the impact of DON and FBs on the intestinal microbiota is species- and age-dependent. Mycotoxin exposure modulated the abundance of several members of the family of Lachnospiraceae, Ruminococcacceae, Peptococaceae, Coriobacteriaceae, Bifidobacteriaceae, Clostridiaceae, and Veillonellaceae. Several of the affected microbial genera are involved in the production of short-chain fatty acids such as butyric acid.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Constantin_2015" class="col-sm-8"> <div class="title">Genetic characterization of strains named as Xanthomonas axonopodis pv. dieffenbachiae leads to a taxonomic revision of the X. axonopodis species complex</div> <div class="author"> E. C. Constantin, I. Cleenwerck, M. Maes, <em>S. Baeyen</em>, C. Van Malderghem, P. De Vos, and B. Cottyn </div> <div class="periodical"> <em>Plant Pathology</em>, Oct 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/ppa.12461" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Rombouts_2015" class="col-sm-8"> <div class="title">Isolation and characterization of Pseudomonas syringae pv. porri from leek in Flanders</div> <div class="author"> S. Rombouts, J. Van Vaerenbergh, A. Volckaert, <em>S. Baeyen</em>, T. De Langhe, B. Declercq, R. Lavigne, and M. Maes </div> <div class="periodical"> <em>European Journal of Plant Pathology</em>, Sep 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1007/s10658-015-0761-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="f75d0742443d475ea999b5f0495fa986" class="col-sm-8"> <div class="title">Effects of biochar addition on bacterial populations of the lettuce and strawberry rhizosphere</div> <div class="author"> Caroline {De Tender}, Jane Debode, Annelies Haegeman, Tom Ruttink, <em>Steve Baeyen</em>, Pieter Cremelie, Bart Vandecasteele, Peter Dawyndt, and Martine Maes </div> <div class="periodical"> Jun 2015 </div> <div class="periodical"> Bageco 13 ; Conference date: 14-06-2015 Through 18-06-2015 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="f949af8d70dc4e7bbaf5a4119931a2b7" class="col-sm-8"> <div class="title">Effects of biochar addition on bacterial populations of the lettuce and strawberry rhizosphere</div> <div class="author"> Caroline {De Tender}, Jane Debode, Annelies Haegeman, Tom Ruttink, <em>Steve Baeyen</em>, Pieter Cremelie, Bart Vandecasteele, Peter Dawyndt, and Martine Maes </div> <div class="periodical"> May 2015 </div> <div class="periodical"> BIG N2N annual symposium ; Conference date: 21-05-2015 Through 21-05-2015 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="b61384bec25d43dbb93e379398d99417" class="col-sm-8"> <div class="title">Towards population genomics and metagenomics in agriculture and marine environments</div> <div class="author"> Annelies Haegeman, Thijs {De Mulder}, Caroline {De Tender}, Marie-Alice Fraiture, Ilia Kirov, Gil Luypaert, Frederik Parijs}, Elisabeth Veeckman, Christophe Verwimp, <em>Steve Baeyen</em>, Jane Debode, Ellen {De Keyser}, Jan {De Riek}, Lisa Devriese, Karen Goossens, Sara Maes, Hilde Muylle, Nico Peiren, Geertrui Rasschaert, Isabel Taverniers, Els {Van Coillie}, Leen Vandaele, Katrijn {Van Laere}, Sam {De Campeneere}, Marc {De Loose}, Marc Heyndrickx, Martine Maes, Johan Robbens, Isabel Roldán-Ruiz, Johan {Van Huylenbroeck}, and Tom Ruttink </div> <div class="periodical"> May 2015 </div> <div class="periodical"> BIG N2N annual symposium ; Conference date: 21-05-2015 Through 21-05-2015 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="cbd05bc146864e8689446bfcbfbcb920" class="col-sm-8"> <div class="title">Metagenome analysis of a soil and marine environment</div> <div class="author"> Caroline {De Tender}, Jane Debode, Lisa Devriese, Annelies Haegeman, Tom Ruttink, <em>Steve Baeyen</em>, Pieter Cremelie, Sara Maes, Peter Dawyndt, Martine Maes, and Johan Robbens </div> <div class="periodical"> May 2014 </div> <div class="periodical"> 4th MRP N2N Bioinformatics symposium ; Conference date: 21-05-2014 Through 21-05-2014 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Za_uga_2014" class="col-sm-8"> <div class="title">Comparative genome analysis of pathogenic and non-pathogenic Clavibacter strains reveals adaptations to their lifestyle</div> <div class="author"> Joanna Załuga, Pieter Stragier, <em>Steve Baeyen</em>, Annelies Haegeman, Johan Van Vaerenbergh, Martine Maes, and Paul De Vos </div> <div class="periodical"> <em>BMC Genomics</em>, May 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1471-2164-15-392" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="7e022d7551e149199ff1ef38f7b341f1" class="col-sm-8"> <div class="title">DGGE fingerprinting for microbial community monitoring as a new tool for anthropogenic impact assessment on the Belgian part of the North Sea</div> <div class="author"> Lisa Devriese, Sara Maes, Caroline {De Tender}, <em>Steve Baeyen</em>, Karen Bekaert, Pieter Cremelie, and Johan Robbens </div> <div class="periodical"> May 2014 </div> <div class="periodical"> Studiedag Zandwinning ; Conference date: 20-10-2014 Through 20-10-2014 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="446239c760dd49ecb31b739ea8500c32" class="col-sm-8"> <div class="title">Population genomics @ ILVO - Plant Sciences Unit</div> <div class="author"> Annelies Haegeman, Tom Ruttink, Hilde Muylle, Christophe Verwimp, Frederik Parijs}, Sabine {Van Glabeke}, Isabel Roldán-Ruiz, Jane Debode, <em>Steve Baeyen</em>, Caroline {De Tender}, and Martine Maes </div> <div class="periodical"> May 2014 </div> <div class="periodical"> Agricultural -Omics course (European Bioinformatics Institute) ; Conference date: 17-02-2014 Through 21-02-2014 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="b1fc446c654a48ff89676f31fa7df633" class="col-sm-8"> <div class="title">ISSR-PCR fingerprinting of plant pathogen strains using the QIAxcel\textregistered system and BioNumerics\textregistered</div> <div class="author"> <em>Steve Baeyen</em>, Joanna Zaluga, Paul {De Vos}, and Martine Maes </div> <div class="periodical"> Apr 2013 </div> <div class="periodical"> Qiagen Application Note </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A reliable method for genotypic fingerprinting of plant pathogen strains would be an important tool for tracking sources of infections and determining effective control strategies. This application note describes a reliable method for this purpose using the QIAxcel Advanced System in combination with BioNumerics software. The plant pathogen model used was bacterial canker of tomato (Clavibacter michiganensis subsp. michiganensis).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Pritchard_2013" class="col-sm-8"> <div class="title">Draft Genome Sequences of Four Dickeya dianthicola and Four Dickeya solani Strains</div> <div class="author"> Leighton Pritchard, Sonia Humphris, <em>Steve Baeyen</em>, Martine Maes, Johan Van Vaerenbergh, John Elphinstone, Gerry Saddler, and Ian Toth </div> <div class="periodical"> <em>Genome Announcements</em>, Aug 2013 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1128/genomea.00087-12" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vandroemme_2013" class="col-sm-8"> <div class="title">Draft genome sequence of Xanthomonas fragariaereveals reductive evolution and distinct virulence-related gene content</div> <div class="author"> Joachim Vandroemme, Bart Cottyn, <em>Steve Baeyen</em>, Paul De Vos, and Martine Maes </div> <div class="periodical"> <em>BMC Genomics</em>, Nov 2013 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1471-2164-14-829" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="376c8f06c632445083fc3b7e99a5d71b" class="col-sm-8"> <div class="title">Draft genome sequencing of Xanthomonas fragariae reveals reduced genome size and distinct virulence related gene content</div> <div class="author"> Joachim Vandroemme, <em>Steve Baeyen</em>, Bart Cottyn, Paul {De Vos}, and Martine Maes </div> <div class="periodical"> Jul 2012 </div> <div class="periodical"> 4th<em> Xanthomonas </em>Genomics Conference ; Conference date: 09-07-2012 Through 12-07-2012 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Xanthomonas fragariae (Xf) is a bacterial strawberry pathogen and an A2 quarantine organism on strawberry planting stock in the EU. It is known for its rather weak pathogenicity, fastidious growth on artificial media, its narrow host range and its taxonomically distinct but homogenous position within the otherwise complex genus Xanthomonas. As part of a broader pathogenicity study of Xf, the genome of a recent Belgian isolate (LMG 25863) was sequenced and assembled to draft status, with 105 contigs of more than 200 bp long. Remarkably, the Xf draft genome (4.2 Mb) was considerably smaller than most other Xanthomonas genomes (~5 Mb). Compared to other plant pathogenic Xanthomonas, Xf appeared to miss several genes/regions with a possible relevance to pathogenicity: i) an operon involved in the breakdown of phenolic compounds, ii) one of the two Type II Secretion System coding regions in Xanthomonas, iii) isocitrate lyase and malate synthase, responsible for the glyoxylate shunt pathway, and iv) two genes of the rfp gene cluster, rpfD and rpfI. Also, only half of the cellulolytic, hemicellulolytic and pectinolytic enzyme coding genes typically present in other Xanthomonas genomes were found in Xf. On the other hand, we identified possible virulence-related gene regions in Xf that are not common in other Xanthomonas genomes, with a Type VI Secretion System coding region being the most obvious one. Also, the Type III Secretion System effector repertoire in the Xf genome included several rare variable effectors such as xopC, xopE4, xopAF and xopAS, and some putatively new effectors. The Xf draft genome further revealed a high content of externally derived DNA: i) multiple copies of several Insertion Sequence (IS) elements, comparable to Xanthomonas oryzae, ii) a phage coding region, iii) 18-19 kb of sequence fragments with high similarity to Plasmid III (27 kb) of X. albilineans GPE GP73, and iv) a CRISPR associated gene cluster of the Yersinia pestis subtype with 40 related spacer sequences. The draft genome sequence of LMG 25863 affirms the distinct position of Xf within the genus Xanthomonas and reveals a patchwork of both lost and newly acquired pathogenicity related functions. It’s pathogenicity related distinguishing features may help explain the specific, mostly endophytic association of Xf with the strawberry plant system.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ef74f7ebb6c7471db3cf8b37f7034ad6" class="col-sm-8"> <div class="title">Xanthomonas arboricola genomics and type three effector gene distribution</div> <div class="author"> {Joël F.} Pothier, J. Blom, <em>Steve Baeyen</em>, S. Essakhi, Ahmed Hajri, Sophie Cesbron, Joachim Vandroemme, Bart Cottyn, A. Goesmann, Martine Maes, Charles Manceau, Theo Smits, and Brion Duffy </div> <div class="periodical"> Jul 2012 </div> <div class="periodical"> 4th<em> Xanthomonas </em>Genomics Conference ; Conference date: 09-07-2012 Through 12-07-2012 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Xanthomonas arboricola pv. pruni (Xap) causes bacterial spot on commercial, ornamental and forest Prunus species. Outbreaks can result in severe economic losses to fruit quality and yield, branch/tree dieback, and orchard devastation, particularly on peach, apricot, nectarine and plum. This stone is endemic in the USA, NZ and is a quarantine regulated pathogen in Europe and elsewhere, compounding economic loss with eradication and regulatory expenses. Wide host range and climate change patterns portend potential increased invasion risk of this pathogen. Despite the regulatory and economic significance of Xap, almost nothing is known about the genetics of Xap compared to other xanthomonads. To fill this gap, a genome sequencing and comparative genomics approach were undertaken. We have sequenced the complete genome of a genotypic-representative Xap strain from Europe (Italy, CFBP 5530). This is the first complete genome sequence for this species. Paired-end 454-pyrosequencing and primer walking on a fosmid library gave 3 contigs. The chromosome is 4.85 Mb with 65.6% GC ratio and 3,912 predicted CDS. Xap has a unique 41.2 kb plasmid with a 62.3% GC ratio and 45 CDS. Comparative genomics with other Xanthomonads was used to identify X. arboricola-specific genes indicating features relevant to differential host specificity and virulence mechanisms. Among those, composition and role of type III secretion system effectors repertoires was surveyed within all six X. arboricola pathovars. Applied genomics has identified over 90 VNTR, several of which are currently being used to examine biodiversity and epidemiology of Xap and related pathovars, and design of improved diagnostics. This first complete genome sequence was also successfully used as a template to perform genomic assembly via mapping of short reads of several isolates from related pathovars. For example, the genome of a Xap isolate GBBC 2038 from the ornamental P. laurocerasus was assembled into 93 contigs. These two genomes show very high identity, the main difference being the presence of 10 kb genomic island in the ornamental isolate. Applied and comparative genomics improved our knowledge on host specificity and biodiversity of Xap, and allowed the development of efficient methods for detection of this plant disease. Overall, the availability of the first genome of X. arboricola constitutes a precious knowledge base for better comprehension of all other X. arboricola pathovars.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="acf0235fd93a45249e8ece0b84fb04ed" class="col-sm-8"> <div class="title">Xanthomonas arboricola genomics and type three effector gene distribution</div> <div class="author"> {Joël F.} Pothier, <em>Steve Baeyen</em>, Jochem Blom, S. Essakhi, Ahmed Hajri, Sophie Cesbron, Joachim Vandroemme, Bart Cottyn, A. Goesmann, Martine Maes, Charles Manceau, {Theo H.M.} Smits, and Brion Duffy </div> <div class="periodical"> Feb 2012 </div> <div class="periodical"> First International Congress for Bacterial Diseases of Stone Fruits and Nuts ; Conference date: 14-02-2012 Through 17-02-2012 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Xanthomonas arboricola pv. pruni (Xap) causes bacterial spot on commercial, ornamental and forest Prunus species. Outbreaks can result in severe economic losses to fruit quality and yield, branch/tree dieback, and orchard devastation, particularly on peach, apricot, nectarine and plum. This stone is endemic in the USA, NZ and is a quarantine regulated pathogen in Europe and elsewhere, compounding economic loss with eradication and regulatory expenses. Wide host range and climate change patterns portend potential increased invasion risk of this pathogen. Despite the regulatory and economic significance of Xap, almost nothing is known about the genetics of Xap compared to other xanthomonads. To fill this gap, a genome sequencing and comparative genomics approach were undertaken. We have sequenced the complete genome of a genotypic-representative Xap strain from Europe (Italy, CFBP 5530). This is the first complete genome sequence for this species. Paired-end 454-pyrosequencing and primer walking on a fosmid library gave 3 contigs. The chromosome is 4.85 Mb with 65.6% GC ratio and 3,912 predicted CDS. Xap has a unique 41.2 kb plasmid with a 62.3% GC ratio and 45 CDS. Comparative genomics with other Xanthomonads was used to identify X. arboricola-specific genes indicating features relevant to differential host specificity and virulence mechanisms. Among those, composition and role of type III secretion system effectors repertoires was surveyed within all six X. arboricola pathovars. Applied genomics has identified over 90 VNTR, several of which are currently being used to examine biodiversity and epidemiology of Xap and related pathovars, and design of improved diagnostics. This first complete genome sequence was also successfully used as a template to perform genomic assembly via mapping of short reads of several isolates from related pathovars. For example, the genome of a Xap isolate GBBC 2038 from the ornamental P. laurocerasus was assembled into 97 contigs. These two genomes present very high identity, the main difference being the presence of 10 kb genomic island in the ornamental isolate. Applied and comparative genomics improved our knowledge on host specificity and biodiversity of Xap, and allowed the development of efficient methods for detection of this plant disease. Overall, the availability of the first genome of X. arboricola constitutes a precious knowledge base for better comprehension of all other X. arboricola pathovars.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Van_Vaerenbergh_2012" class="col-sm-8"> <div class="title">Sequence Diversity in the Dickeya fliC Gene: Phylogeny of the Dickeya Genus and TaqMan® PCR for “D. solani”, New Biovar 3 Variant on Potato in Europe</div> <div class="author"> Johan Van Vaerenbergh, <em>Steve Baeyen</em>, Paul De Vos, and Martine Maes </div> <div class="periodical"> <em>PLoS ONE</em>, May 2012 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1371/journal.pone.0035738" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Cottyn_2010" class="col-sm-8"> <div class="title">Development of a real‐time PCR assay for Pseudomonas cichorii, the causal agent of midrib rot in greenhouse‐grown lettuce, and its detection in irrigating water</div> <div class="author"> B. Cottyn, <em>S. Baeyen</em>, E. Pauwelyn, I. Verbaendert, P. De Vos, P. Bleyaert, M. Höfte, and M. Maes </div> <div class="periodical"> <em>Plant Pathology</em>, Nov 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/j.1365-3059.2010.02388.x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Alaei_2009" class="col-sm-8"> <div class="title">Molecular detection of Puccinia horiana in Chrysanthemum x morifolium through conventional and real-time PCR</div> <div class="author"> Hossein Alaei, <em>Steve Baeyen</em>, Martine Maes, Monica Höfte, and Kurt Heungens </div> <div class="periodical"> <em>Journal of Microbiological Methods</em>, Feb 2009 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1016/j.mimet.2008.10.001" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Debode_2009" class="col-sm-8"> <div class="title">Quantitative detection and monitoring of Colletotrichum acutatum in strawberry leaves using real‐time PCR</div> <div class="author"> J. Debode, W. Van Hemelrijck, <em>S. Baeyen</em>, P. Creemers, K. Heungens, and M. Maes </div> <div class="periodical"> <em>Plant Pathology</em>, May 2009 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/j.1365-3059.2008.01987.x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Bobev_2008" class="col-sm-8"> <div class="title">First Record of Bacterial Blight Caused by Pseudomonas syringae pv. syringae on Pyracantha coccinea and an Amelanchier sp. in Bulgaria</div> <div class="author"> S. G. Bobev, <em>S. Baeyen</em>, J. Van Vaerenbergh, and M. Maes </div> <div class="periodical"> <em>Plant Disease</em>, Aug 2008 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/pdis-92-8-1251c" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vandroemme_2008" class="col-sm-8"> <div class="title">Sensitive real‐time PCR detection of Xanthomonas fragariae in strawberry plants</div> <div class="author"> J. Vandroemme, <em>S. Baeyen</em>, J. Van Vaerenbergh, P. De Vos, and M. Maes </div> <div class="periodical"> <em>Plant Pathology</em>, Feb 2008 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1111/j.1365-3059.2007.01813.x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2006</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Crepel_2006" class="col-sm-8"> <div class="title">First Report of Cochliobolus sativus on Guzmania sp. in Belgium</div> <div class="author"> C. Crepel, S. Inghelbrecht, <em>S. Baeyen</em>, and M. Maes </div> <div class="periodical"> <em>Plant Disease</em>, Oct 2006 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/pd-90-1361a" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2005</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Crepel_2005" class="col-sm-8"> <div class="title">First Report of Leaf Spots on Laurus nobilis Caused by Phytophthora citricola in Belgium</div> <div class="author"> C. Crepel, S. Inghelbrecht, <em>S. Baeyen</em>, M. Maes, and S. G. Bobev </div> <div class="periodical"> <em>Plant Disease</em>, Jan 2005 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/pd-89-0107b" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2004</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Bobev_2004" class="col-sm-8"> <div class="title">First Report of Fire Blight Caused by Erwinia amylovora on Pyracantha coccinea in Bulgaria</div> <div class="author"> S. G. Bobev, <em>S. Baeyen</em>, C. Crepel, and M. Maes </div> <div class="periodical"> <em>Plant Disease</em>, Apr 2004 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/pdis.2004.88.4.427a" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2003</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Steve_Baeyen_19524329" class="col-sm-8"> <div class="title">Experiences and perspectives for the use of a Paenibacillus strain as a plant protectant.</div> <div class="author"> <em>Steve Baeyen</em>, Maes M, and Baeyen S </div> <div class="periodical"> <em></em> Apr 2003 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Baeyen2003" class="col-sm-8"> <div class="title">Experiences and perspectives for the use of a Paenibacillus strain as a plant protectant.</div> <div class="author"> M. Maes and <em>S. Baeyen</em> </div> <div class="periodical"> <em>Communications in agricultural and applied biological sciences</em>, Apr 2003 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Bobev_2003" class="col-sm-8"> <div class="title">First Report of Phytophthora cactorum on American Ginseng (Panax quinquefolius) in Bulgaria</div> <div class="author"> S. G. Bobev, <em>S. Baeyen</em>, C. Crepel, and M. Maes </div> <div class="periodical"> <em>Plant Disease</em>, Jun 2003 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1094/pdis.2003.87.6.752c" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Steve Baeyen. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. Last updated: July 26, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>